TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+          15811626,
+          20654738,
+          24334031,
+          28089915,
+          15774093,
+          20836881,
+          20739849,
+          22133464,
+          22321138,
+          22321138,
+          20704716,
+          20911153,
+          22656843,
+          23445874,
+          14896892,
+          12343580,
+          22037020,
+          23884809,
+          23029213,
+          21841232,
+          21337518,
+          16009583,
+          28086477,
+          19345315,
+          18585146,
+          25244974,
+          23877257,
+          22394733,
+          20614959,
+          20083786,
+          15048037,
+          28531069,
+          23143560,
+          24616128,
+          20604712,
+          24614028,
+          27539205,
+          38783854,
+          16714699,
+          17841891,
+          29244150,
+          28348062,
+          25117617,
+          25634843,
+          27038225,
+          22786507,
+          22786507,
+          22905607,
+          25993467,
+          27591483,
+          30085953,
+          40993951,
+          18766305,
+          22494434,
+          18260267,
+          27722511,
+          26350323,
+          21736748,
+          21392376,
+          21730051,
+          26640595,
+          26640595,
+          28476232,
+          26940627,
+          33851356,
+          23951914.29,
+          23951914.29,
+          31485172.45,
+          26894099.17,
+          28842985.58,
+          28706805.95,
+          28477729.82,
+          28923675.96,
+          27125138.45,
+          25496118,
+          25496118,
+          24062535.05,
+          24062535.05,
+          23009994.04,
+          21495765.02,
+          25903635.94,
+          35280351.26,
+          29567891.31,
+          25872048.71,
+          21686611.83,
+          21686611.83,
+          21078311.83,
+          23822896.59,
+          27675629.09,
+          27675629.09,
+          34447048.43,
+          35186711.31,
+          41151034.39,
+          32862919.91,
+          33603536.36,
+          32793984.32,
+          29082683.33,
+          29082683.33,
+          27899472.03,
+          26613186.99,
+          30239538.42,
+          28628692.39,
+          31147315.59,
+          27685184.24,
+          27685184.24,
+          27685184.24,
+          31954759.69,
+          42596195.27,
+          37729078.44,
+          36994026.79,
+          36994026.79,
+          38813251.1,
+          39431767.13,
+          39653551.79,
+          40285883.82,
+          37330027.69,
+          37230401.33,
+          36501888.28,
+          39865469.32,
+          41012369.65,
+          37530273.1,
+          27159271.45,
+          27159271.45,
+          34204242.15,
+          35291615.88,
+          34725461.34,
+          35439598.96,
+          34493610.47,
+          35987055.01,
+          29894777.05,
+          29894777.05,
+          30098714.42,
+          30098714
```

Comment:
I've not reviewed this at all; but, two things are needed.
1. So far my reorg of what you sent uses 60 minutes for review and then demo. This may or may not be appropriate, I'll let that be something you consider during your revisions; but, assuming it was the right time allotments, this would leave you 40 minutes for a communication activity.
    1. Look at weeks 1-4 TUTs (and weeks 8 and 9 are also pretty finalized as another good example) to see what this can look like. Personally, I wouldn't make it a writing activity... I would make it some sort of verbal communication oriented activity... writing can be moved to the homework.
2. We need a HW assignment here. Look at weeks 1-4 TUTs (and weeks 8 and 9 are also pretty finalized as another good example) to see what this can look like.

Reply:
It's supposed to be a comment on 12091-12107:

"## Tutorial Assignment [0 minutes]\n",
     "\n",
     "### Part 1:\n",
     "You have just been selected as Canada's Finance Minister with the expectation that you will implement effective policies that will promote economic growth in the country. You are quite proficient in both financial and statistical analysis and you decide that regression analysis can be particularly useful to determine what your fiscal policy should be. You are required to present a report to a committee involving senior officials of the country to justify the measures you are suggesting should be taken to significantly boost Canada's GDP.\n",
     "<br><br>\n",
     "\n",
     "In your report of 250-350 words, you must explain the significance of regression analysis, identify (with justification) one variable (such as years of schooling, retirement age etc.) which you believe directly impacts and has a very strong correlation with economic growth. Include an equation of the relationship between the economic growth variable and the predictor variable you have identified. You must communicate in a clear and professional manner.\n",
     "\n",
     "### Part 2:\n",
     "Whilst making a 'cheat-sheet' for your STA130 mid-term, you were relying on ChatGPT for assistance, but you realized that there were several errors with some of the ideas presented by the AI engine. Now, your job is to develop a critical mindset so you can detect when information online or given by an AI engine lacks credibility. Ask ChatGPT the following questions; if you believe the responses are correct, paraphrase them, or identify the mistake otherwise.\n",
     "\n",
     "1. What are indicator variables and what is their importance in regression analysis?\n",
     "2. What is the difference between the 'expected value' and 'model predicted value' in linear regression?\n",
     "3. How can we determine if a linear regression model is a good fit for a dataset?\n",
     "\n",
     "For each question, show ChatGPT's answer and write a small paragraph underneath. Compile your responses in a Word Document and submit on Quercus. "